CSS/main.css

```diff
@@ -5,18 +5,33 @@
 }
 
 body {
-    background-image: linear-gradient(100deg, #575656, #062e3f);
     /* #363532, rgb(88, 111, 112) */
-    color: #ffdfdb;
     font-family: 'Work Sans', sans-serif;
     min-height: 100vh;
-    margin-top: 70px;
+    padding-top: 70px;
+}
+
+/* Body light or darker themes */
+.standard {
+    background-image: linear-gradient(100deg, #575656, #062e3f);
+    color: #ffdfdb;
+}
+
+.light {
+    background-image: linear-gradient(100deg, #EBF1F1, #B6BBBF);
+    color: #1a150e;
+}
```

Comment:
I was trying different colors, for the light theme, and I thought this is going well:

![Screenshot (763)](https://user-images.githubusercontent.com/61280281/98965119-ac56ea80-252f-11eb-88c2-55727f4d13df.png)

`    background-image: linear-gradient(100deg, #d4f1ff, #ffffff);
`

Reply:
Looks fine to me too, I'll change the light one to it! The different themes I chose were only suggestions, there's absolutely no problem for me changing them a bit! ;)